holidays/countries/nepal.py

```
@@ -0,0 +1,441 @@
+
+
+class NPL(Nepal):
+    pass
+
+
+class NepalStaticHolidays:
+    """Nepal special holidays.
+
+    References:
+        * [Death of Krishna Prasad Bhattarai](https://myrepublica.nagariknetwork.com/news/27618/)
+        * [Death of Sushil Koirala](https://kathmandupost.com/miscellaneous/2016/02/09/cabinet-decision-koirala-to-be-honoured-with-state-funeral)
```

Pylint (3.3.7)

[convention] Line too long (146/100)

(C0301)